sqldelight-gradle-plugin/src/test/fixtures/function-name-tests/src/main/sqldelight/com/sample/Capitals.sq

```diff
@@ -0,0 +1,3 @@
+capitals_work_fine:
+SELECT COUNT(*), MAX(some_int)
+FROM test;
```

Comment:
yes. is the fixtures runner smart enough to know that there were failures in one file, but a different file succeeded and it should check for generated code?